ast/builtins.go

```diff
@@ -2327,6 +2328,40 @@ with all others being treated as intermediates.`,
 	),
 }
 
+var CryptoX509ParseAndVerifyCertificatesWithOptions = &Builtin{
+	Name: "crypto.x509.parse_and_verify_certificates_with_options",
+	Description: `Returns one or more certificates from the given string containing PEM
+or base64 encoded DER certificates after verifying the supplied certificates form a complete
+certificate chain back to a trusted root. A config option passed as the second argument can 
+be used to configure the validation options used.
+
+The first certificate is treated as the root and the last is treated as the leaf,
+with all others being treated as intermediates.
+
+` +
+		"`options` object has the same fields as [x509.VerifyOptions struct](https://pkg.go.dev/crypto/x509#VerifyOptions). " +
```

Comment:
This gives the impression that we support all the `VerifyOptions` fields which is not the case, correct? 
We can just mention the values we support and point to the Go docs for more info. We can skip the below description as well.

Reply:
Made the change.